.s2i-dev/bin/usage

```diff
@@ -4,7 +4,7 @@ cat <<EOF
 This is a S2I release-bot image:
 To use it, install S2I: https://github.com/openshift/source-to-image
 Sample invocation:
-s2i build <configuration_repository> release-bot release-bot-app
+s2i build <configuration_repository> usercont/release-bot:dev release-bot-app:dev
```

Comment:
Instead of `usercont` use `user-cont`, right? In the whole PR.

Reply:
Uppss, usercont is image name and not GitHub repository.

Reply:
[usercont](https://hub.docker.com/u/usercont) is our organization on docker hub. The image is  [usercont/release-bot](https://hub.docker.com/r/usercont/release-bot)